.github/workflows/long_faucet_chain_test.yml

```diff
@@ -16,7 +16,7 @@ env:
 jobs:
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +6]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->You need to do it for this one too<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/3018?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->